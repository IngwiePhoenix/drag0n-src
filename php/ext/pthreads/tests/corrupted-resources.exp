Work Object
(
)
object(Work)#3 (0) {
}
resource(2) of type (stream)
resource(2) of type (stream)