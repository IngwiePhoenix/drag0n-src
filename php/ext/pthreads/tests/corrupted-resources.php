<?php
class Work extends Stackable {
	public function run(){
		
	}
}

class Test extends Thread {
	public function run(){
		$test = new Work();
		$this->test = $test;
		print_r($this->test);
		var_dump($this->test);
		$stream = fopen("/tmp/test.txt", "w+");
		var_dump($stream);
		$this->stream = $stream;
		stream_set_blocking($this->stream, 0);
		var_dump($this->stream);
	}
}

$test =new Test();
$test->start();
?>
