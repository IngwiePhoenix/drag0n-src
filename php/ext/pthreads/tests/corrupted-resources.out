Warning: Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) in Unknown on line 0
Work Object
(
)
object(Work)#3 (0) {
}
resource(2) of type (stream)
resource(2) of type (stream)